content/deploy/backup/influx.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Can we set to this is a `draft` until we have updated the installation to include influxdb?